src/test/scala/com/microsoft/hyperspace/index/plananalysis/ExplainTest.scala

```diff
@@ -518,8 +521,14 @@ class ExplainTest extends SparkFunSuite with HyperspaceSuite {
     verifyExplainOutput(df, expectedOutput.toString, verbose = false) { filterQuery }
   }
 
-  private def getIndexFilesPath(indexName: String): Path = {
+  private def getIndexRootPath(indexName: String): Path =
     new Path(systemPath, s"$indexName/v__=0")
+
+  private def getIndexFilesPath(indexName: String): Path = {
+    val path = getIndexRootPath(indexName)
+    val fs = path.getFileSystem(new Configuration)
+    // Pick any files path but remove the _SUCCESS file.
```

Comment:
We should reuse `isDataPath`?

Reply:
oh like by either convert `isDataPath` to public method or add it to `PathUtils`? I can do that. Thanks

Reply:
thanks @imback82 , done

Reply:
added this as util to PathUtils.